recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "transformers" %}
-{% set version = "4.32.1" %}
+{% set version = "4.37.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 1edc8ae1de357d97c3d36b04412aa63d55e6fc0c4b39b419a7d380ed947d2252
+  sha256: 5a0fdee36168f751770f7036ce7a8787be14f8b0d8f29806c493b6cb819c6c83
 
 build:
   skip: True  # [py<37]
```

```suggestion
  skip: True  # [py<38]
```
https://github.com/huggingface/transformers/blob/v4.37.2/setup.py#L447